packages/core/src/Cipher.ts

```diff
@@ -0,0 +1,16 @@
+import { Service } from 'typedi';
+import { AES, enc } from 'crypto-js';
+import { InstanceSettings } from './InstanceSettings';
+
+@Service()
+export class Cipher {
```

Comment:
`EncryptionService`? Afaik `Cipher` is the specific algorithm, but the main concern of this class is encrypting and decrypting.

Reply:
I find `Cipher` to be a generic term for something that does encryption and decryption.